main.tf

```diff
@@ -803,7 +803,8 @@ resource "aws_iam_role_policy" "xosphere_terminator_policy" {
       ],
       "Resource": "arn:*:autoscaling:*:*:autoScalingGroup:*:autoScalingGroupName/*",
       "Condition": {
-        "StringEquals": {"autoscaling:ResourceTag/xosphere.io/instance-orchestrator/enabled": "true"}
+        "StringEquals": {"autoscaling:ResourceTag/xosphere.io/instance-orchestrator/enabled": "true"},
+        "StringEquals": {"autoscaling:ResourceTag/xosphere:instance-orchestrator:enabled": "true"}
```

Comment:
This is an `and`, not an `or`.  We'll need to split these into separate SIDs.

Reply:
Oops. I think we can just use a single StingEquals with multiple elements, no?